resources/definitions/os_discovery/routeros.yaml

```diff
@@ -242,3 +242,24 @@ modules:
                     num_oid: .1.3.6.1.4.1.14988.1.1.6.1.{{ $index }}
                     descr: DHCP Lease Count
                     index: mtxrDHCPLeaseCount.{{ $index }}
+                -
+                    oid: MIKROTIK-MIB::mtxrCtTotalEntries
+                    num_oid: .1.3.6.1.4.1.14988.1.1.22.1.1.{{ $index }}
+                    descr: 'Total number of connections'
+                    index: mtxrCtTotalEntries.{{ $index }}
+                    group: Firewall
+                    skip_value_lt: 1
```

Comment:
Do we really want to skip values of 0? The sensors will get removed at the point of discovery if they are 0 which I expect they could be.

Reply:
> Do we really want to skip values of 0? The sensors will get removed at the point of discovery if they are 0 which I expect they could be.

mi original intention was to hide Firewall counters if they are not used.
Owning a MT router does not mean you using firewall/tracking

but yes, you are right
overthinking this time :)